bot/rodhaj.py

```diff
@@ -83,6 +83,10 @@ def stop():
             self.logger.info("Dev mode is enabled. Loading FSWatcher")
             self.loop.create_task(self.fs_watcher())
 
+    def user_is_this_bot(self, user: Union[discord.User, discord.Member]):
```

Comment:
just check the attr of `discord.User`'s bot

for example:

```py
# Assume we have the user variable, which is the type of discord.User
return user.bot
```

in reality you probably don't even need this helper method to begin with

Reply:
Deleted this method